docs/volcano_integration.md

```diff
@@ -0,0 +1,107 @@
+# Integration with Volcano for Batch Scheduling
+
+[Volcano](https://github.com/volcano-sh/volcano) is a batch system built on Kubernetes. It provides a suite of mechanisms
+currently missing from Kubernetes that are commonly required by many classes
+of batch & elastic workloads.
+With the integration with Volcano, Flink job and task managers can be scheduled for better scheduling efficiency.
+
+## Prerequisites
+
+## Install Volcano
+
+- Install from provided demo
+
+Run the following 
+```bash
+kubectl apply -f https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml
+```
+
+- Install with advanced settings
+
+Please refer to [Volcano Official Guide](https://volcano.sh/docs/getting-started/)
+ 
+## Install Flink Operator with Volcano enabled
+
+## Installing the Chart
```

Comment:
Can we support customizing `--enable-batch-scheduler` for `make deploy` as well? You can just follow what we do for `--watch-namespace`.

Reply:
Will look into